### ISSUE TITLE
'evince' is not redognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by Sinooshka** [geany/geany#3745]
I have Geany 2.0 on windows 10. I have successfuly compiled my .tex file but am unable to create the corresponding .pdf  (program exited with code: 9009)
I tried to manually add the address of Evince to the PATH variable. The problem did not solve. 
I would appreciate any comment on it.

**Comment by elextr** [comment]
Can you run evince from the command line?

**Comment by Sinooshka** [comment]
No.

**Comment by elextr** [comment]
Then its not going to run from inside Geany, you need to fix that issue first.

**Comment by b4n** [comment]
FWIW, evince is a document *viewer*, it's not gonna help *create* PDFs. And if you already have a PDF viewer you can use it instead as well.

**Comment by elextr** [comment]
Well "compile" means `pdflatex` to me, which "just does" PDF and is in most tex distros IIRC.

**Comment by b4n** [comment]
Well I read "but am unable to create the corresponding .pdf", so I am not exactly sure which step doesn't work.

**Comment by elextr** [comment]
> Well I read "but am unable to create the corresponding .pdf", so I am not exactly sure which step doesn't work.

Good point, I read evince and thought the OP had the PDF, but now I'm not sure either.

**Comment by elextr** [comment]
Oh wait, the default latex filetype file has both `latex` and `pdflatex` as default build commands and `evince` as the "execute" command, but its probably not available on Windows, which is why the OP can't run it from the command line either.

**Comment by Sinooshka** [comment]
Sorry, maybe my explanation was not clear. The .tex file is successfully
compiled. That is the corresponding .pdf is already created but as you
mentioned 'evince' is not being recognized by geany/windows therefore the
process of View PDF File(F5) is not working. I tried to install SumatraPDF.
But I do not know how to change the syntax in the corresponding "Set Build
Commands" for geary to successfully run F5 and show the .pdf

**Comment by Sinooshka** [comment]
I think I found a way. I added the line bellow to the "Execute Commands"
section for viewing .pdf files:

start ""/max "%e.pdf"

Thanks again for your comments, and sorry if I was not clear.
